vagabond/app/views/users/show.html.erb

```diff
@@ -1,20 +1,35 @@
-<div>
-<p>First Name: <%=  @user.first_name %> </p>
-<p>Last Name: <%=  @user.last_name %> </p>
-<p>Email: <%= @user.email %> </p>
-<p>Date Joined: <%= @user.created_at.localtime.strftime("%b %-d, %Y %H:%M") %> </p>
-<p>Current City: <%=  @user.current_city %> </p>
+<div class="col-xs-3">
+  <div class="row">
+    <div class="col-xs-10">
+      <% if @user.image_url == nil%>
```

Comment:
Idiomatic ruby would be either:
* [x] `if @user.image_url` # i.e string is truthy
* [ ] `unless @user.image_url.nil?` # but I'm not a big fan of unless